src/managing-workflow/production-deployments.md

```diff
@@ -1,3 +1,57 @@
 # Production Deployments
 
-TODO (bacongobbler): rewrite for v2
+When readying a Workflow deployment for production workloads, there are some additional (but
```

Comment:
Since they are recommendations, you could omit "(but optional)" as it's redundant.